automatic/displayfusion/displayfusion.nuspec

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>displayfusion</id>
+    <version>0.0</version>
+    <title>DisplayFusion</title>
+    <authors>Binary Fortress Software</authors>
+    <projectUrl>http://www.displayfusion.com</projectUrl>
+    <licenseUrl>https://www.displayfusion.com/License/</licenseUrl>
+    <iconUrl>https://images.binaryfortress.com/ChocolateyIcons/DisplayFusion.png</iconUrl>
```

Comment:
Put icon in the icon repository of this package as per [§1.1.5](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#115-provide-icon). Make sure to use commit hash as per [§1.5.2](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#152-use-commit-hash).